File: Python/government.py
from utility import Utility
from gla import Gla
import sys

class Government(Gla, Utility):
    def __init__(self):
        super().__init__()
        self.governor = "Shri. P. S. Sreedharan Pillai"
        self.cm = "Shri. Pramod Sawant"
        self.com_array = [
            "SHRI. PRAMOD SAWANT",
            "SHRI. VISHWAJIT RANE",
            # ... (all other ministers)
            "SHRI. ALEIXO SEQUEIRA"
        ]
        self.com = "\n\n".join(self.com_array)

    def set_details(self):
        menuchoice = 100
        while menuchoice != 0:
            print()
            print("=============================== *** GOA GOVERNMENT DETAILS *** ===============================")
            print("=============================== *** ENTER YOUR CHOICE *** ===============================")
            print()
            print("1] GOVERNOR \t\t\t 2] CHIEF MINISTER \t\t\t 3] COUNCIL OF MINISTERS")
            print()
            print("4] POLITICAL GROUPS \t\t\t")
            print()
            print("=============================== *** ENTER 0 TO EXIT *** ===============================")
            print()
            self.print_assembly_year()
            menuchoice = int(input("=> ").strip())

            if menuchoice == 1:
                print(f"GOVERNOR FOR THE ASSEMBLY IS {self.governor}")
            elif menuchoice == 2:
                print(f"CHIEF MINISTER OF GOA IS {self.cm}")
            elif menuchoice == 3:
                print(f"COUNCIL OF MINISTERS ARE {self.com}")
            elif menuchoice == 4:
                print(f"POLITICAL GROUPS ARE \n\n{self.political_groups}")
            elif menuchoice == 0:
                sys.exit()
            else:
                print("PLEASE ENTER VALID INPUT")

File: C#/GLA_APP/Government.cs
// Government.cs
using System;

public class Government : Gla, IUtility
{
    private string Governor = "Shri. P. S. Sreedharan Pillai";
    private string CM = "Shri. Pramod Sawant";
    private string[] COM_Array = 
    {
        "SHRI. PRAMOD SAWANT",
        "SHRI. VISHWAJIT RANE",
        "SHRI. MAUVIN GODINHO",
        "SHRI. RAVI NAIK",
        "SHRI. SUBHASH SHIRODKAR",
        "SHRI. ROHAN KHAUNTE",
        "SHRI. GOVIND GAUDE",
        "SHRI. ATANASIO MONSERRATE",
        "SHRI. RAMKRISHNA ALIAS SUDIN DHAVALIKAR",
        "SHRI. NILKANTH HALARNKAR",
        "SHRI. SUBHASH PHAL DESSAI",
        "SHRI. ALEIXO SEQUEIRA"
    };
    private string COM => string.Join("\n\n", COM_Array);

    public void SetDetails()
    {
        int menuChoice = 100;
        while (menuChoice != 0)
        {
            Console.WriteLine();
            Console.WriteLine("=============================== *** GOA GOVERNMENT DETAILS *** ===============================");
            Console.WriteLine("=============================== *** ENTER YOUR CHOICE *** ===============================");
            Console.WriteLine();
            Console.WriteLine("1] GOVERNOR \t\t\t 2] CHIEF MINISTER \t\t\t 3] COUNCIL OF MINISTERS");
            Console.WriteLine();
            Console.WriteLine("4] POLITICAL GROUPS \t\t\t");
            Console.WriteLine();
            Console.WriteLine("=============================== *** ENTER 0 TO RETURN TO MAIN MENU *** ===============================");
            Console.WriteLine();
            PrintAssemblyYear();
            Console.Write("=> ");
            menuChoice = int.Parse(Console.ReadLine());

            switch (menuChoice)
            {
                case 1:
                    Console.WriteLine($"GOVERNOR FOR THE ASSEMBLY IS {Governor}");
                    break;
                case 2:
                    Console.WriteLine($"CHIEF MINISTER OF GOA IS {CM}");
                    break;
                case 3:
                    Console.WriteLine($"COUNCIL OF MINISTERS ARE {COM}");
                    break;
                case 4:
                    Console.WriteLine($"POLITICAL GROUPS ARE \n\n{PoliticalGroups}");
                    break;
                case 0:
                    return;
                default:
                    Console.WriteLine("PLEASE ENTER VALID INPUT");
                    break;
            }
        }
    }
}
